api/admin/password_admin_authentication_provider.py

```diff
@@ -8,18 +8,20 @@
     Session,
 )
 from problem_details import *
+from template_styles import *
 
 class PasswordAdminAuthenticationProvider(AdminAuthenticationProvider):
 
     NAME = "Password Auth"
 
     TEMPLATE = """
+
 <form action="%(password_sign_in_url)s" method="post">
 <input type="hidden" name="redirect" value="%(redirect)s"/>
-<label>Email <input type="text" name="email" /></label>
-<label>Password <input type="password" name="password" /></label>
-<button type="submit">Sign In</button>
-</form>"""
+<label style="{}">Email <input type="text" name="email" style="{}" /></label>
+<label style="{}">Password <input type="password" name="password" style="{}" /></label>
+<button type="submit" style="{}">Sign In</button>
+</form>""".format(label_style, input_style, label_style, input_style, button_style)
```

Comment:
This would be more readable and more resistant to change if you passed key-value pairs into `format` instead of a list:

`<label style="{label}">Email <input type="text" name="email" style="{input}" /></label>`
`.format(label=label_style, input=input_style, button=button_style)`